zircon.core/src/commonTest/kotlin/org/hexworks/zircon/TestFunctions.kt

```diff
@@ -7,3 +7,10 @@ fun TileComposite.fetchCharacters(): List<Char> {
         getTileAt(it).get().asCharacterTile().get().character
     }
 }
+
+/**
+ * Ensure each line is at least [length] long, padding it with [padChar] if it's not. This splits the string on newline
+ * internally and uses [lineSeparator] to re-join the lines.
+ */
+fun String.padLineEnd(length: Int, padChar: Char = ' ', lineSeparator: String = "\n"): String =
```

Comment:
I'm not sure if there is a constant for the line separator that works on both Linux and Windows. In this case this will only work for Linux because on Windows a line separator also contains a carriage return (`\r`).

Reply:
IIRC I specifically wanted "\n" because that's the kind that AssertJ uses internally. You _can_ use the system one here, but then I think you have to use one of the lengthy `assertThat(...).isEqualToNormalizingLineEndings(...)` calls.

Reply:
Ok, we can leave it like this.